### DISCUSSION TITLE
Development continuity?

### DISCUSSION THREAD

**Discussion by tjsuominen** [tommasoberlose/another-widget#390] [General]
Just wondering if the development still going to evolve at some point? Love AW but some enhancements could be made to catch up Google 'At a glance' :) Hope all is well with the dev!

**Comment by Shanaoth** [comment]
Would be nice to add weather.com provider to match the Google weather page ;)

**Comment by Sarkis-ix** [comment]
I like your widget, nice and minimalistic. But, I would like to ask you for a few dorobotki.
1. Add your own custom fonts
2. Ability to add a font to the clock
3. Add weather opening integration with default weather app.
4. Add weather provider AccuWeather

**Comment by Plazzmex** [comment]
I'm not sure the OG developer is still here, he didn't post any update in the last 2 years, hope everything is ok with him. Anyway, Check out Azuo fork. He added a lot of fixes and new stuff since the latest official version

https://github.com/azuo/another-widget

**Reply by tjsuominen** [reply]
Appreciated, thx.

**Reply by anup514** [reply]
Thanks a lot!
